index.html

```diff
@@ -96,6 +96,7 @@ <h2 class="block-h2">Multi-Post Stories</h2>
                         </ul>
                     </div>
                     <button class="interaction right-btn">See Project</button>
```

Comment:
[ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). kindly fix that 🤓